apps/web/pages/organizations/[organizationId]/invoices/new.page.tsx

```diff
@@ -23,7 +24,7 @@ const NewInvoicePage = (): JSX.Element => {
   const router = useRouter()
 
   const { organizationId } = router.query
-  const invoiceDate = new Date().toString()
+  const invoiceDate = format(new Date(), 'yyyy-MM-dd').toString()
```

Comment:
```suggestion
  const invoiceDate = format(new Date(), 'yyyy-MM-dd')
```
I did not try but is the .toString() required here ?

Reply:
Correct, it's not required. The format takes care of it already.